src/fetch-info.js

```diff
@@ -68,6 +68,19 @@ async function useLastInfoForDiscontinuedSpecs(specs) {
   return results;
 }
 
+function catchAndFallbackOnExistingData(fn) {
+  return function(spec) {
+    return fn(spec).catch(err => {
+      if (spec.__last) {
+	// TODO: log crawl error?
```

Comment:
We really need to log an error somewhere, be it to the console. Otherwise we won't even be able to tell that we're reusing the last information. That's a very good way to hide the problem, but means we could get stuck with hidden build failures for months.